lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -5,6 +5,7 @@ import 'package:sticky_headers/sticky_headers.dart';
 class DSBottomSheetService {
   final BuildContext context;
   final Widget child;
+  final Widget childAppBar;
```

Comment:
`childAppBar` is not declared in the constructor

Reply:
Adjusted.